data/transportation/transit/index.html

```diff
@@ -6,205 +6,189 @@
+  type="GIS Data Layer"
+%}
 <div class="grid package">
-    <div class="grid__col grid__col--12-of-12">
-        <h3 id="UTABusRoutes&Stops">UTA Bus Routes & Stops</h3>
-    </div>
-    <div class="grid__col grid__col--12-of-12 package-content">
-<img class="productImage-Thumb" src="{{ "/images/BusStops_Small.png" | prepend: site.baseurl }}" alt="Changeme" />Transportation.BusRoutes_UTA includes bus stops along streets, at TRAX stations, and park & ride lots serviced by UTA buses. Transportation.BusStops_UTA includes redesigned bus routes that have been adjusted for internal monitoring purposes and ski bus routes to the Cottonwood Canyons and Sundance Ski Resort.
-The UTA Bus Stops dataset was digitized to the Wasatch Front counties' street base maps and then exported from Trapeze (scheduling software) in shapefile format.</p>
-Questions can be directed to <a href="mailto:lnelms@rideuta.com">Lorraine Nelms</a> (801.237-1947) at UTA.[/tab]</p>
-Download UTA Bus Routes & Stops:</p>
-<ul>
-<li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mXy0wcENfR0NWN0k&export=download">Bus Routes: File Geodatabase</a></li>
-<li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mRU9XR3FuTWhDMzA&export=download">Bus Routes: Shapefile</a></li>
-<li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mRTJwVU52c1VCNDA&export=download">Bus Stops: File Geodatabase</a></li>
-<li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mMF9ZUkdiMEpsb3M&export=download">Bus Stops: Shapefile</a></li>
-</ul>
-<p>Other links for this dataset:</p>
-<ul class="productLinks">
-<li class="productLink"><a href="ftp://ftp.agrc.utah.gov/SGID93_Vector/NAD83/MetadataHTML/SGID93_TRANSPORTATION_BusRoutes_UTA.html">View FGDC format metadata (Bus Routes)
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="UTABusRoutesStops">
+      <a href="#UTABusRoutesStops" class="anchor--link">UTA Bus Routes & Stops</a>
+    </h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="productImage-Thumb" src="{{ '/images/BusStops_Small.png' | prepend: site.baseurl }}" alt="Bus Stops" />
+    <p>Transportation.BusRoutes_UTA includes bus stops along streets, at TRAX stations, and park & ride lots serviced by UTA buses. Transportation.BusStops_UTA includes redesigned bus routes that have been adjusted for internal monitoring purposes and ski bus routes to the Cottonwood Canyons and Sundance Ski Resort.</p>
+    <p>The UTA Bus Stops dataset was digitized to the Wasatch Front counties' street base maps and then exported from Trapeze (scheduling software) in shapefile format.</p>
+    <p>Questions can be directed to <a href="mailto:lnelms@rideuta.com">Lorraine Nelms</a> (801) 237-1947 at UTA.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li class="productLink">
```

Comment:
you can remove all usage of `class=productLink`. I prefer to remove the `View` piece as well